reservation_units/migrations/0034_reservationunit_tax_percentage.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1.13 on 2021-12-08 09:59
+
+from decimal import Decimal
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('reservation_units', '0033_reservationunit_remove_price'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='reservationunit',
+            name='tax_percentage',
+            field=models.DecimalField(choices=[(Decimal('0.00'), '0%'), (Decimal('10.00'), '10%'), (Decimal('14.00'), '14%'), (Decimal('24.00'), '24%')], decimal_places=2, default=Decimal('24.00'), help_text='The tax percentage for the price of the reservation unit', max_digits=5, verbose_name='Tax percentage'),
+        ),
+        migrations.AlterField(
+            model_name='reservationunit',
+            name='reservation_start_interval',
```

Comment:
I think we have this missing from`main` => perhaps separate PR with separate migration file.

Reply:
Nicely spotted. 🙂 Fixed now and added a new PR for that.